SWIG/montecarlo.i

```diff
@@ -156,7 +156,31 @@ class GaussianMultiPathGenerator {
       }
     }
     Sample<MultiPath> next() const;
-	Sample<MultiPath> antithetic() const;
+    Sample<MultiPath> antithetic() const;
+};
+
+%{
+typedef QuantLib::MultiPathGenerator<GaussianLowDiscrepancySequenceGenerator>
+    GaussianMultiPathSobolGenerator;
+%}
+class GaussianMultiPathSobolGenerator {
```

Comment:
I think this one and the existing one can be written as a single template, like `PathGenerator` in this same file?

Reply:
Thanks - let me Investigate!